mods/3d_armor/3d_armor/crafting_guide.txt

```diff
@@ -17,8 +17,8 @@ Helmets:
 [3d_armor:helmet_bronze] X = [default:bronze_ingot]
 [3d_armor:helmet_diamond] X = [default:diamond]
 [3d_armor:helmet_gold] X = [default:gold_ingot]
-[3d_armor:helmet_mithril] X = [default:mithril_ingot]
-[3d_armor:helmet_crystal] X = [ethereal:crystal_ingot] *
+[3d_armor:helmet_mithril] X = [moreores:mithril_ingot] *
```

Comment:
This line should not be changed to `moreores`, `default:mithril_ingot` is still the item name for mithril ingots on MFF, since we merged moreores into default ages ago.

Reply:
More generally, consider not changing this later in the file as well.

Reply:
Should I keep it as is, or change it?

Reply:
I think you should, just in case.

Reply:
Okay.